MIP40/MIP40c3-Subproposals/MIP40c3-SP90.md

```diff
@@ -7,13 +7,14 @@ MIP40c3-SP#: 90
 Author(s): @Patrick_J, @prose11
 Contributors:
 Tags: budget, cu-budget, gov-001
-Status: Formal Submission
+Status: Obsolete
 Date Applied: 2023-02-07
-Date Ratified: <yyyy-mm-dd>
+Date Ratified: N/A
 Budget Start Date: 2023-04-01
 Budget End Date: 2024-03-31
 Forum URL: https://forum.makerdao.com/t/mip40c3-sp90-govalpha-core-unit-dai-budget-2023-2024/19712
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmXoH4rE
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```

Comment:
For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.